ci/jobs/scripts/check_style/check_cpp.sh

```diff
@@ -348,6 +348,15 @@ do
     echo "Found the usage of std::format in '${file}'. Please use fmt::format instead"
 done
 
+# Forbid using quotes for includes (except for autogenerated files)
+QUOTE_EXCLUSIONS=(
+    --exclude "$ROOT_PATH/utils/memcpy-bench/glibc/*"
+)
+find $ROOT_PATH/{programs,utils} -name '*.h' -or -name '*.cpp' | xargs grep -P '#include[\s]*(").*(")' "${QUOTE_EXCLUSIONS[@]}" | grep -v config\.h | grep -v config_tools\.h | while read -r line;
+do
+   echo "Found include with quotes in '${line}'. Please use <> instead"
+done
```

Comment:
```suggestion
find $ROOT_PATH/{programs,utils} -name '*.h' -or -name '*.cpp' | xargs grep -P '#include[\s]*(").*(")' "${QUOTE_EXCLUSIONS[@]}" | grep -v -F -e config.h -e config_tools.h | xargs -i echo "Found include with quotes in '{}'. Please use <> instead"
```

Slightly simpler alternative, but yours also OK